src/components/SubjectTierListing/SubjectTierListing.tsx

```diff
@@ -3,35 +3,17 @@ import React, { FC } from "react";
 import TierList from "../TierList";
 import TitleCard from "../Card/SubjectUnitLessonTitleCard";
 import { Heading } from "../Typography";
-import { TierListingData } from "../../node-lib/curriculum-api";
+import { ProgrammeListingPageData } from "../../node-lib/curriculum-api-2023/queries/programmeListing/programmeListing.schema";
 
-type ProgrammeDetails = {
-  keyStageTitle: string;
-  keyStageSlug: string;
-  subjectTitle: string;
-  subjectSlug: string;
-};
-
-type SubjectTierListingProps = {
-  programmeDetails: ProgrammeDetails;
-};
-
-const SubjectTierListing: FC<SubjectTierListingProps & TierListingData> = ({
-  programmeDetails,
-  programmes,
-}) => {
-  const { keyStageTitle, keyStageSlug, subjectTitle, subjectSlug } =
-    programmeDetails;
-  const tierTrackingProps = {
-    keyStageTitle,
-    subjectTitle,
-  };
+const SubjectTierListing: FC<ProgrammeListingPageData> = ({ ...props }) => {
+  const { programmes, keyStageSlug, subjectSlug, keyStageTitle = "" } = props;
+  const subjectTitle = programmes[0]?.subjectTitle ?? "";
 
   return (
     <>
       <TitleCard
         page={"subject"}
-        keyStage={keyStageTitle}
+        keyStage={keyStageTitle ?? ""}
```

Comment:
This was left out of the MV - i'm going to add it back in here:
https://www.notion.so/oaknationalacademy/I-want-to-connect-the-programme-listing-MV-to-the-2023-curriculum-API-ebc22f117c434ef4bb917ece325fe1c7